### PR TITLE
add new fhir pattern for cql eval

### PR DESCRIPTION
**Pull request by ranum**
Signed-off-by: David Ranum <dlranum@us.ibm.com>

### CODE REVIEW

File: services/ibm-fhir-server/fhir-cql-eval/pom.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.ibm.fhir</groupId>
+	<artifactId>fhir-ibmcloud</artifactId>
+	<version>99-SNAPSHOT</version>
```
Comment (ibmatc) [inline services/ibm-fhir-server/fhir-cql-eval/pom.xml:7]: Is this version exposed anywhere? Should it be something other than a snapshot?

**Review by ibmatc — APPROVED** [review]
LGTM

File: services/ibm-fhir-server/fhir-cql-eval/pom.xml (hunk 1 above)
Comment (lmsurpre) [reply services/ibm-fhir-server/fhir-cql-eval/pom.xml:7]: if david updates it to match what I did at https://github.com/Alvearie/health-patterns/blob/main/services/ibm-fhir-server/data-access/Dockerfile then the pom can be dropped entirely

Comment (ranum) [reply services/ibm-fhir-server/fhir-cql-eval/pom.xml:7]: Yeah, not sure.  I was just following the pattern that was in the data-access pom

**Pull request merged**